packages/components/src/internal/components/search/FindSamplesByIdsPageBase.tsx

```diff
@@ -106,11 +120,23 @@ export const FindSamplesByIdsTabbedGridPanelImpl: FC<FindSamplesByIdsTabProps> =
                     alwaysShowTabs: true,
                     getAdvancedExportOptions,
                     exportFilename: 'SamplesFoundById',
-                    getGridPanelDisplay,
-                    getShowViewMenu
+                    showViewMenu: false,
                 }}
                 user={user}
             />
+            {printDialogModel && allSamplesModel && (
```

Comment:
I had envisioned PrintLabelsModal will be shown/hide by SamplesTabbedGridPanel, without having to control it by individual caller pages. Is there a reason printDialogModel can't be part of SamplesTabbedGridPanel?

Reply:
Yes, I agree that we should move towards putting the label printing modal code directly into the SamplesTabbedGridPanel component so that we don't need to duplicate that implementation for each usage. I was chatting with @labkey-susanh about this earlier today (re: if/when we are doing to enabled label printing from Source > Samples, Picklist > Samples, Job > Samples, etc.). 

That said, I think we should leave this as is for this PR to get things moving through the pipeline to make sure we get this merged in for 22.9 and then tackle the SamplesTabbedGridPanel part in a separate PR. Susan added a note on that to the LKB/LKSM app consistency spec, so it won't be lost.